playbooks/roles/postgresql_aggregation/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+- name: Install packages
+  apt:
+    name: "{{ item }}"
+    install_recommends: yes
+    state: present
+  with_items: postgresql_aggregation_packages
+
+- name: install requirements for role
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items: postgresql_aggregation_requirements
+
+- name: check if user already exists
+  command: "psql -h {{ postgresql_aggregation_login_host }} -U {{ postgresql_aggregation_login_user }} -d postgres -c \"SELECT 1 FROM pg_roles WHERE rolname='{{ POSTGRESQL_AGGREGATION_USER }}'\""
```

Comment:
@melvinsoft One thing I wonder about: Is this task really needed, or will the postgresql_user task just skip if the user has already been created?